lib/OpenQA/WebSockets.pm

```diff
@@ -71,11 +71,13 @@ sub ws_send {
 
     my $tx = $worker->{tx};
     if (!$tx || $tx->is_finished) {
+        # uncoverable statement untested exceptional error
         log_debug("Unable to send command \"$msg\" to worker $workerid: worker not connected");
```

Comment:
Might be better to actually test this? At least that ~error~ log message looks like something important :thinking:

Reply:
Well, are you interested in extending the test? The problem is that t/27-websockets.t does not cover much of this file yet at all. But you can give it a shot if you like. Otherwise I would keep it as is because I think it's fine if this exceptional error branch is not covered and we except it as such.

Reply:
I would like to see that we fix that test and not "forget about it" because it no longer shows up, but no reason to block this one. So yes, but approving.